install-gns3.sh

```diff
@@ -293,36 +297,17 @@ $my_separator
     check_for_gns3_server=$(pacman -Qe | grep -c python-gns3-server)
     if [[ $check_for_gns3_server -lt 1 ]]; then
       printf %b\\n "${On_Red}
-    GNS 3 Server was not installed...
+    GNS3-server was not installed...
 
       attempting to re-install${Color_Off}"
       PKGEXT=.pkg.tar pypi2pkgbuild.py -g cython -b /tmp/pypi2pkgbuild/ -f git+file://"$HOME"/GNS3-Dev/gns3-server
       sleep 1
     fi
   else
     printf %b\\n "${IGreen}
-  Everything looks alright. If you are using Gnome or Budgie, the script can make a launcher icon for you.
-
-  1 = yes, 2 = no${Color_Off}"
-    read -r make_launcher
-    if [[ $make_launcher == 1 ]]; then
-      sudo tee -a /usr/share/applications/gns3.desktop >/dev/null <<EOL
-[Desktop Entry]
-Type=Application
-Encoding=UTF-8
-Name=GNS3
-GenericName=Graphical Network Simulator 3
-Comment=Graphical Network Simulator 3
-Exec=/usr/bin/gns3
-Icon=gns3
-Terminal=false
-Categories=Application;Network;Qt;
-EOL
-    fi
-    printf %b\\n "${IGreen}
```

Comment:
Any specific reason to remove this part? I use gnome and find it quite handy - I'm sure others might as well. A better solution can be to customize and offer a choice depending on a DE the user is running. Not a top priority for me, but something I could work on in the near future.

Reply:
I am on KDE and an icon was created automatically for me, so during this part of the script it complained about the file already existing.
Maybe GNS3 changed its installation process? Or this is something KDE specific.